### ISSUE TITLE
Update contributing guide

### ISSUE THREAD

**Issue by peterdesmet** [tdwg/dwc#167]
[Contributing guide](https://github.com/tdwg/dwc/blob/master/.github/CONTRIBUTING.md) should include:

* What is part of the standard and what is not (an implementation of SDS)
* What the rules are for updating the standard (an implementation of VMS)
* How one can submit an issue
* For the adventurous: how one can update the `term_versions.csv` vocabulary as a pull request: add new line, update previous term

And we should include an ISSUE_TEMPLATE.md. As we can only have one, I think a term change or new term is probably the most used one...

**Comment by tucotuco** [comment]
The document currently resides at https://github.com/tdwg/dwc/blob/master/.github/CONTRIBUTING.md. Would it be better at https://github.com/tdwg/dwc? 

I agree that there should be a reference to the VMS, but is this really the place to say what is normative in the standard?

I would recommend against adding anything about amending term_versions.csv.

I set up two issue templates, one for new term requests and one for term change requests.

If I can get consensus I'll proceed to make the changes.

**Comment by tucotuco** [comment]
Partial changes made to match issue templates in https://github.com/tdwg/dwc/commit/551c64eb2d93cddf49987452cf84e3a24f4ecaaa

**Comment by baskaufs** [comment]
With respect to the usage comments, we might indicate that they are optional and only necessary if the definition does not provide all of the needed information about how the term should be used. I don't know how to say that in a way that is not too wordy and suitable for a form.

**Comment by tucotuco** [comment]
A lot of this content is in the [FAQ](https://github.com/tdwg/dwc/wiki/Darwin-Core-Maintenance-Frequently-Asked-Questions), which was created primarily for the latest public review. We should have a strategy for merging and placing this material. Best in an .md file for consistency probably. Recommendations welcome.

**Comment by tucotuco** [comment]
Updated the contributing guide and integrated the Wiki FAQ with it.
